move/sources/hongbao.move

```diff
@@ -517,10 +519,16 @@ module addr::hongbao {
         caller_address: address,
         amount: u64
     ) {
-        if (object::object_address(&gift_metadata) == @0xa) {
+        let fa_metadata_address = object::object_address(&gift_metadata);
+        // for hongbao and APT, only transfer coin to avoid many new FA's going around
+        if (fa_metadata_address == @0xa) {
             aptos_account::transfer_coins<0x1::aptos_coin::AptosCoin>(
                 signer, caller_address, amount
             );
+        } else if (fa_metadata_address
+            == @0x180e877b151107d7d180545c2fdb373578740872a59071f636c62bd60a6b249d) {
```

Comment:
Shouldn't we be using @emoji_coin instead.

Reply:
this is the fungible asset address

Reply:
@emoji_coin is the coin address